testsuite/MDAnalysisTests/coordinates/test_chemfiles.py

```diff
@@ -40,9 +41,9 @@
 chemfiles = pytest.importorskip("chemfiles")
 
 
-@pytest.mark.parametrize('version', ['0.9.3', '0.11.0', '1.1.0'])
+@pytest.mark.parametrize("version", ["0.9.3", "0.11.0", "1.1.0"])
 def test_version_check(version, monkeypatch):
-    monkeypatch.setattr('chemfiles.__version__', version)
+    monkeypatch.setattr("chemfiles.__version__", version)
```

Comment:
As far as I can tell (please correct me if I'm right) these ' to " don't adhere to any given convention, why are they all being changed? (i.e. is this worth the git history  footprint)

Reply:
It's `black`'s convention: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#strings

Reply:
Exactly, these were automated changes by black. I can revert them if you prefer.

Reply:
I'm not a fan of arbitrary (read non-PEP/NEP/*EP) line changes, especially if they affect the history. However, I will leave it to you to decide on what to do here.

Reply:
I tend to treat `black` as mostly standard nowadays, especially given it is developed by the PSF, and the output is quite stable.

Reply:
Actually had a long discussion yesterday with @RMeli (or more we have the same long discussion every couple of months).

My stance is a pretty simple yet strong one re: `black`'s "style": if the PSF wants to codify it as a PEP or even a BEP, then I'd be happy adopting it & evangelizing it. Otherwise it remains a set of decisions set by an arbitrary tool choice, which makes it hard for me to go "no you should do X not Y".

In any case, given you're the code author of the lines being changed, not a hill I wish to spend more time on.